.idea/DevOps-Course-Starter.iml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The .idea folder should probably be placed in a .gitignore.

Also worth mentioning that we'll be pushing for people to use VScode going forward (so at the very least please make sure that it can be run from VScode without additional setup)